src/core/build_simulations.jl

```diff
@@ -17,7 +17,7 @@ function _prepare_workspace!(ref::SimulationRef, base_name::AbstractString, fold
     ref.raw = raw_ouput
     ref.models = models_json_ouput
     ref.results = results_path
-
+    cd(current_working_directory)
```

Comment:
This code block seems fishy.  Shouldn't it be `if !isdir`?  Seems like you could do this without changing directories.